libr/anal/esil_cfg.c

```diff
@@ -578,13 +578,13 @@ R_API RAnalEsilCFG *r_anal_esil_cfg_op(RAnalEsilCFG *cfg, RAnal *anal, RAnalOp *
 	}
 	RAnalEsilBB *glue_bb = R_NEW0 (RAnalEsilBB);
 	if (!glue_bb) {
-		eprintf ("Couldn't allocate glue_bb\n");
+		R_LOG_ERROR ("Couldn't allocate glue_bb");
```

Comment:
imho we shouldnt have any eprintf() on allocation fails. otherwise r2 will be full of different error strings for something that should not ever happen. and if it does, the error should be handled upstream.
```suggestion
```